docker/dockerfile-base

```diff
@@ -0,0 +1,48 @@
+FROM alpine:latest
+
+ENV ELM_VERSION 0.19.1
+ENV SCALA_VERSION 2.12.6
+ENV SBT_VERSION 1.2.1
+
+RUN apk add --no-cache git curl bash openjdk8
+
+# ELM
+RUN curl -L -o elm.gz https://github.com/elm/compiler/releases/download/$ELM_VERSION/binary-for-linux-64-bit.gz && \
+  gunzip elm.gz && \
+  chmod +x elm && \
+  mv elm /usr/local/bin/
+
+# SBT (Scala)
+ENV PATH /sbt/bin:$PATH
+RUN wget https://piccolo.link/sbt-$SBT_VERSION.tgz && \
+  tar -xzvf sbt-$SBT_VERSION.tgz && \
+  rm sbt-$SBT_VERSION.tgz
+#  sbt sbtVersion
+
+# DOCSPELL
+RUN mkdir /src && \
+  git -C /src clone https://github.com/eikek/docspell
+
+
+RUN SBT_OPTS="-Xms1024M -Xmx8G -Xss2M -XX:MaxMetaspaceSize=8G" && \
```

Comment:
I think, if there is the `-mem 4096` option, all these options shouldn't have any noticeable effect. I think that the `-mem` option overrides these settings (or many of them) anyways. I tested without them, and the image was created successfully (on my machine at least).